src/projectscene/view/clipsview/selectionviewcontroller.cpp

```diff
@@ -24,7 +24,7 @@ void SelectionViewController::onPressed(double x, double y)
 
     if (!selectionController()->isDataSelected()) {
         TrackIdList tracks = determinateTracks(m_startPoint.y(), y);
-        selectionController()->setDataSelectedOnTracks(tracks, true);
+        selectionController()->setSelectedTracks(tracks, true);
```

Comment:
I think this method might be simplified to
```
void SelectionViewController::onPressed(double x, double y)
{
    m_selectionStarted = true;
    m_startPoint = QPointF(x, y);
    emit selectionStarted();

    selectionController()->resetDataSelection();
    TrackIdList tracks = determinateTracks(m_startPoint.y(), y);
    selectionController()->setSelectedTracks(tracks, true);
}
```
which would also make it behave more like AU3. Say I have two tracks selected, and I click on one of them to drag my selection. In Audacity 3, this will deselect the non-clicked track, while here it won't.
Or was this a new spec I'm not aware of?